static/style.css

```diff
@@ -81,16 +71,21 @@ div#loading-bar-fill {
   background-color: blue;
 }
 
-form#start {
-  margin: 0 auto;
-}
-
 button#start {
   font-size: 2em;
   padding: 0.75em 1.25em;
   font-weight: bold;
 }
 
+table tr > td:first-child {
+  text-align: left;
+}
+
+table tr > td {
+  text-align: center;
+  padding: 0 1em;
+}
```

Comment:
Make the rightmost columns right aligned

Reply:
Implemented in 706c426